### DISCUSSION TITLE
Elgato Stream Deck

### DISCUSSION THREAD

**Discussion by VAsHachiRoku** [LibreHardwareMonitor/LibreHardwareMonitor#1394] [Ideas]
Would love an Elgato Stream deck plugin. Today I use HWInfo64 to monitor CPU, Memory, GPU plus set thresholds that change colors on the buttons!

**Comment by jlewisnj** [comment]
There is a plugin for Open Hardware Monitor for the stream deck and since LibreHardwareMonitor is a fork from it, the plugin seems to work well for me..
https://github.com/atcurtis/streamdeck-ohs

![Sample](https://github.com/user-attachments/assets/37f848e8-b150-4e40-b17a-22fef6aaf2cb)

**Comment by moeilijk** [comment]
There's now a plugin built directly on LHM that does what you describe — CPU, memory and GPU on keys, with thresholds that change the button colors: https://github.com/moeilijk/lhm-streamdeck (I'm the author).

It connects through LHM's remote web server on port 8085, so LHM itself needs no modification. Also on the Elgato Marketplace: https://marketplace.elgato.com/product/libre-hardware-monitor-af576388-8cbb-4d59-bdec-206dc3f4168e

**Reply by Blacktempel** [reply]
I'd suggest working with LHM master / pre-releases for your software/repository instead of postponing until a new release.

Releases can take months+ and are not guaranteed to have fully stable code, as you can already see.
You also get fresh functionality as it becomes available.
